packages/mobile/src/screens/favorites-screen/FavoritesScreen.tsx

```diff
@@ -41,13 +53,20 @@ export const FavoritesScreen = () => {
     FeatureFlags.OFFLINE_MODE_ENABLED
   )
 
+  const savedTracks = useProxySelector(getTracks, [])
+
   return (
     <Screen>
       <ScreenHeader
         text={messages.header}
         icon={IconFavorite}
         styles={{ icon: { marginLeft: 3 } }}
-      />
+      >
+        <DownloadToggle
```

Comment:
Why not have `DownloadToggle` get the tracks from the store (instead of passing them in)?

Reply:
The same toggle component will need to get tracks from the current collection or from favorites. I figured it'd be easier to pass it in from the parent than to bake in the knowledge of where to select it from state based on `collection_id | 'favorites'`